Applications/ResNet/gen_input/convert.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+#
+# @file	convert.py
+# @date	28 Jan 2021
+# @brief	This is script to convert cifar10 data set for resnet model
+# @see		https://github.com/nnstreamer/nntrainer
+# @author	Jijoong Moon <jijoong.moon@samsung.com>
+# @bug		No known bugs except for NYI items
+#
+# python3 convert.py path_for_cifar10 [train | val]
+#  - trainingSet.dat for training : 100 images x 10 classes
+#  - valSet.dat for validation : 20 images x 10 classes
+#
+
+import os
+import sys
+import imageio
+import numpy as np
+import random
+
+from PIL import Image
+
+num_val = 20
+num_train = 100
+num_class = 10
+
+##
+# @brief extract_features from image
+#
+def extract_features(path, index, val):
+    label_ = ["airplane", "automobile", "bird", "cat", "deer",
+    "dog", "frog", "horse", "ship", "truck"]
+    
+    label_name = np.array(label_)
+
+    path = path + "/train/"
+    class_id =0
+
+    if(val == True):
+        class_id = index // num_val
+        path = path + (label_name[class_id])+'/'+ '{:04d}'.format(500-(index % num_val))+'.bmp'
+    else:
+        class_id = index // num_train
+        path = path + (label_name[class_id])+'/'+ '{:04d}'.format((index % num_train)+1)+'.bmp'
+
+    image = Image.open(path)
+    data = np.array(image)
+    data = np.transpose(data, [2,0,1])
+
+    label = np.zeros(num_class, dtype=np.uint8)
```

Comment:
Simple question:
It is okay to use np.uint8 for dtype?
Cause getData function in main.cpp the size of num_class is treated as float but the range of np.uint8 is 0 to 255